www/doc/2.1/api-endpoints.html

```diff
@@ -31,9 +31,8 @@ <h2 id="all-endpoints-info">Format for all endpoints</h2>
     otherwise specified. A typical example of sending a POST request
     to a running Herbie server is:</p>
 
-    <pre class="shell">curl -X POST -d \
-  '{"formula": "(FPCore (x) (- (sqrt (+ x 1))))",\
-    "seed": 5})}' \
+    <pre class="shell">curl -d \
+  '{"formula": "(FPCore (x) (- (sqrt (+ x 1))))", "seed": 5}' \
```

Comment:
Is -X POST the default in -d mode?

Reply:
Yep, stack overflow told me. Removed it trying to figure out why it wasn't working.

Here is a link to the docs which say that -d implies a POST request will be made.

https://curl.se/docs/manpage.html#-d